airbyte-webapp-e2e-tests/cypress/pages/settingsPage.ts

```diff
@@ -0,0 +1,23 @@
+export const goToSettingsSourcesTab = () => {
+  cy.intercept("/api/v1/source_definitions/list_for_workspace").as("getSourceDefinitionsList");
+  cy.visit("/settings/source");
+  cy.wait(5000);
+};
+
+export const goToSettingsDestinationTab = () => {
+  cy.intercept("/api/v1/destination_definitions/list_for_workspace").as("getDestinationDefinitionsList");
+  cy.visit("/settings/destination");
+  cy.wait(5000);
+};
+
+export const editVersionByConnectorName = (connectorName: string, version: string) => {
+  cy.get("[data-testid='connectorNameCell']")
+    .contains(connectorName) // div with icon and text
+    .parents("tr")
+    .find("[data-testid='version-input']")
```

Comment:
We could change the test id to include the connector name, but I don't see us using dynamic testid's anywhere so went with this.  Finds the div with the connector name, gets its parent row, finds the input within that row.